src/tests/suite/daos_test.c

```diff
@@ -454,7 +454,7 @@ main(int argc, char **argv)
 	}
 
 	if (strlen(tests) == 0) {
-		strcpy(tests , all_tests);
+		strncpy(tests, all_tests, sizeof(TESTS));
```

Comment:
Are you using the right variable here?
```suggestion
		strncpy(tests, all_tests, sizeof(tests));
```

Reply:
Yes. "TESTS" is a macro of a string constant and "all_tests" is set as "TESTS".

https://github.com/daos-stack/daos/blob/master/src/tests/suite/daos_test.c#L27
static const char *all_tests = TESTS;